src/install/config/apache.yaml

```diff
@@ -46,10 +39,21 @@ appInfo:
         recommendedGuided: true        
         logo: 'linux'                  
 
+# ----------------------------------------------------
+# Ask the first question and then provide instructions
+# ----------------------------------------------------
 steps:
+  # Step 1: Ask the user what kind of deployment--not their OS
   - filePath: 'src/install/apache/appInfo.mdx'
+
+  # Step 2: Display the requirements
   - filePath: 'src/install/apache/default-requirements.mdx'
+
+  # Step 3: Display the instructions to install the infra agent (prerequisite)
   - filePath: 'src/install/apache/install-infra.mdx'
+
+  # Step 4: Display the Apache integration download and installation instructions 
+  # based on their answer in question 1 above. Note the overrides:
   - filePath: "src/install/apache/linux/install-apt.mdx"
     overrides: 
         - filePath: "src/install/apache/linux/install-apt.mdx"
```

Comment:
line 57 nests the apt into the default flow for the logic. This is persistent throughout all of these OHIs, and it's why all of them have "APT" int he right nav. To fix this, we need to add a new mdx file that's a "default install" step, then shift `        - filePath: "src/install/apache/linux/install-apt.mdx"` as an option, not the default.

I make this fix in `/install/microsoft-sql.yml` on lines 62-64 for an example. :)